### PR TITLE
[READY] Blind quirk without anger

### PR DESCRIPTION
**Pull request by Thalpy**
Someone got mad and uh... went a little crazy.
So I fixed their PR. I'm not gonna link it because of the name of it.

## About The Pull Request

Adds blind quirk and a blindfold. Also changes crazy eye sprites.

## Why It's Good For The Game

I think their original PR was made in good intent, but they got frustrated with the time it took. People get angry and they absolutely shouldn't have done what they did, and there's no justification in what they did. But it pains me to see people work go to waste, so it's back again with a friendlier face.

Oh god I'm so sleepy

## Changelog
:cl: Fermi
fix: fixed an angery PR
/:cl:

### CODE REVIEW

**Comment by cebutris** [comment]
Fermis is truly a gift to our repository.

**Comment by lolman360** [comment]
Fermis is the only reason I still code for Citadel.

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -281,6 +281,33 @@
 	..()
 	user.cure_blind("blindfold_[REF(src)]")
 
+/obj/item/clothing/glasses/sunglasses/blindfold/white
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/glasses/_glasses.dm:284]: Fun fact cult stuns will still work against blinding people do to being a magic light that appears in their head!

**Comment by Linzolle** [comment]
jesus christ what the hell happened to that last PR. thanks fermis, holy shit

File: code/modules/clothing/glasses/_glasses.dm (hunk 1 above)
Comment (Thalpy) [reply code/modules/clothing/glasses/_glasses.dm:284]: What spell? Do all cult spells rely on sight?

Comment (Trilbyspaceclone) [reply code/modules/clothing/glasses/_glasses.dm:284]: The clocks use a blinding light from clock slab.
And Blood cult use a sorta stun spell as well that is a white flash of light.
No matter what, even well blind you get stunned by these attacks, no cult spell I know of relys on sight nor relys on a targets sight

**Comment by Thalpy** [comment]
Maybe this will fix the false negative.

**Review by silicons — APPROVED** [review]

**Pull request merged**